File: programacion-orientada-a-objetos/CursoPOOUber/Python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial   = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial): #Datos obligatorios para que existe el objeto
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
    
    def setPassenger(self, passenger):
        if passenger == 6:
            self.passenger = passenger
        else:
            self.passenger = None
            print("Debe asignar 6 pasajeros")

File: programacion-orientada-a-objetos/CursoPOOUber/JS/UberVan.js
class UberVan extends Car{
    constructor(license, driver, typeCarAccepted, seatsMaterial){
        super(license, driver)
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    //General functions
    setPassenger(passenger){
        if (passenger == 6) {
            this.passenger == passenger;
        }else{
            console.log("Necesitas asignar 6 pasajeros")
        }
    }
}